core/src/main/java/ai/timefold/solver/core/impl/domain/solution/descriptor/SolutionDescriptor.java

```diff
@@ -727,11 +727,17 @@ private void validateListVariableDescriptors() {
 
         var listVariableDescriptor = listVariableDescriptorList.get(0);
         var listVariableEntityDescriptor = listVariableDescriptor.getEntityDescriptor();
-        if (listVariableEntityDescriptor.getGenuineVariableDescriptorList().size() > 1) {
+        var hasChained = listVariableEntityDescriptor.getGenuineVariableDescriptorList().stream()
```

Comment:
I think this check needs to be smarter. The chained variable can be defined on _any_ entity in the solution.